alkira/resource_alkira_probe_http.go

```diff
@@ -0,0 +1,190 @@
+package alkira
+
+import (
+	"context"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAlkiraProbeHTTP() *schema.Resource {
+	return &schema.Resource{
+		Description:   "Manage Alkira HTTP Probe.",
+		CreateContext: resourceProbeHTTPCreate,
+		ReadContext:   resourceProbeHTTPRead,
+		UpdateContext: resourceProbeHTTPUpdate,
+		DeleteContext: resourceProbeHTTPDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the HTTP probe.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"enabled": {
+				Description: "Whether the probe is enabled.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+			},
+			"network_entity": {
+				Description: "Network entity configuration.",
+				Type:        schema.TypeList,
+				Required:    true,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Description: "The type of network entity to probe." +
+								" Only `INTERNET_APPLICATION` supported for now.",
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"id": {
+							Description: "The ID of the network entity.",
```

Comment:
What ID is this? We never really have term `network entity` used anywhere else before... so it will be helpful to be more specific.